index.bs

```diff
@@ -1043,8 +1048,8 @@ that determine how WebTransport connection is established and used.
    trusted if and only if it can successfully [=verify a certificate hash=] against
    {{WebTransportOptions/serverCertificateHashes}}
    and satisfies [=custom certificate requirements=].  The user agent SHALL
-   ignore any hash that uses an unknown {{RTCDtlsFingerprint/algorithm}}
-   or has a malformed {{RTCDtlsFingerprint/value}}.  If empty, the user agent
+   ignore any hash that uses an unknown {{WebTransportHash/algorithm}}
+   or has a malformed {{WebTransportHash/value}}.  If empty, the user agent
```

Comment:
We'll want to define what "malformed" means now.  I believe that this is - now - only that it is shorter or longer than the output of the selected algorithm.

Reply:
"verify a certificate hash" contains the logic. We ignore _hash_ if
 1. _hash_'s value is null,
 2. _hash_'s value doesnt match with _referenceHash_.

The case where the length doesn't match is contained in 2. If we need to define "malformed" we can remove this sentence - What do you think?

Reply:
WFM.

Reply:
Done.